frontend/app/assets/javascripts/spree/frontend/views/spree/products/index.js

```diff
@@ -108,4 +108,24 @@ Spree.ready(function ($) {
       setNewUrl(searchParams);
     }
   });
+
+  $('#filterPriceRange').on('submit', function(event) {
+    event.preventDefault();
+
+    var $priceRangeForm = $(event.currentTarget);
+    var minPrice = parseInt($priceRangeForm.find('[name="min_price"]').val());
+    var maxPrice = parseInt($priceRangeForm.find('[name="max_price"]').val());
+
+    var searchParams = new URLSearchParams(location.search);
```

Comment:
@mad-eel rather than doing it the JS way we can just set the form URL to `url: permitted_params` and rails will handle the rest :) also this will work better with Turbo(links)

Reply:
@damianlegawiec I'm not sure what you're suggesting here. The problem I see here is that the filtering param looks like this: `$<min_price> - $<max_price>` so just having a form with inputs won't do.

Do you think we can safely alter how we filter by params? And just do it with `min_price` and `max_price`?

Reply:
@mad-eel we can remove the `$` from forms, just numbers will do